manifests/server/kickstart/runpuppet.pp

```diff
@@ -51,8 +96,16 @@
   Optional[Simplib::Host]     $puppet_ca               = simplib::lookup('simp_options::puppet::ca', { 'default_value' => undef }),
   Simplib::Port               $puppet_ca_port          = simplib::lookup('simp_options::puppet::ca_port', { 'default_value' => 8141 }),
   Stdlib::Absolutepath        $location                = '/var/www/ks/runpuppet',
```

Comment:
@lnemsick-simp I think that we should change this to something else by default. We definitely don't want to keep this as `runpuppet` since that may break existing systems.

`simp_client_bootstrap` may be a better name since it is more descriptive for our users.